node/src/BaseClient.ts

```diff
@@ -847,6 +1183,13 @@ export class BaseClient {
      *
      * @param key - The key to return the number of its members.
      * @returns the cardinality (number of elements) of the set, or 0 if key does not exist.
```

Comment:
And a lot of other places
```suggestion
     * @returns The cardinality (number of elements) of the set, or 0 if key does not exist.
```